FanSpeedControl/FanSpeedControl.js

```diff
@@ -112,19 +119,25 @@ const FanSpeedControlBase = kind({
 	},
 
 	computed: {
+		backgroundColor:({skinVariants}) => skinVariants.night ? '#444444' : '#eeeeee',
 		children: ({min, max}) => range(min, max + 1),
-		className: ({icon, styler}) => styler.append({noIcon: !icon})
+		className: ({icon, styler}) => styler.append({noIcon: !icon}),
+		foregroundColor:({skinVariants}) => skinVariants.night ? '#eeeeee' : '#444444'
 	},
 
-	render: ({children, disabled, icon, max, min, onChange, value, ...rest}) => {
+	render: ({backgroundColor, children, disabled, foregroundColor, icon, max, min, onChange, value, ...rest}) => {
 		const ariaProps = extractAriaProps(rest);
 
+		delete rest.skinVariants;
+
 		return (
 			<div {...rest}>
 				<ArcPicker
 					{...ariaProps}
+					backgroundColor={backgroundColor}
 					disabled={disabled}
 					endAngle={312}
+					foregroundColor={foregroundColor}
```

Comment:
Can't we just give the default color in arcPicker without fixing this?
FanSpeedControl and ArcPicker are not the same color?
If you do that, both components will be modified with one code.

Reply:
Ok, I reverted the changes from FanSpeedControl.js

Reply:
Then, will the ArcPicker be modified in other issues? 
I cannot find issue for ArcPicker.

The color has not been corrected.
https://github.com/enactjs/agate/pull/543#issuecomment-801834279

Reply:
I think you missed this comment.

Reply:
Yes, sorry, I missed the comment, I added the default values in ArcPicker. 
Also there is no ticket for ArcPicker because other than the colors there were no other changes requested for it.